src/Features/VisualBasic/Portable/VBFeaturesResources.resx

```diff
@@ -1224,4 +1224,8 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
   <data name="Project_level_Imports" xml:space="preserve">
     <value>Project level 'Imports'</value>
   </data>
+  <data name="Could_not_extract_interface_colon_Cannot_extract_interface_for_modules" xml:space="preserve">
+    <value>Could not extract interface: Cannot extract interface for modules</value>
```

Comment:
phrase seems redundant.  can it just be `Can not extract interface for modules`?

Reply:
It just follows the other phrasing we use. Happy to change to just "Can not extract interface for modules" though if that's preferred

Reply:
if this is consistent with the other strings, then i'm fine keeping as is :)